src/convertToObject.js

```diff
@@ -13,6 +13,19 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const formattedSource = sourceString.split(';')
+    .map(property => {
+      return property.split(':');
+    })
+    .filter(element => element.length === 2)
+    .map(element => element.map(property => property.trim()));
```

Comment:
Твоє форматування тоже ок, просто додатково покажу як би я... :)
```suggestion
    .map(property => property.split(':'))
    .filter(element => element.length === 2)
    .map(element => element
      .map(property => property.trim()));
```